spec/components/candidate_interface/restructured_work_history_review_component_spec.rb

```diff
@@ -0,0 +1,158 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::RestructuredWorkHistoryReviewComponent do
+  around do |example|
+    Timecop.freeze(Time.zone.local(2019, 10, 1)) do
+      example.run
+    end
+  end
+
+  let(:february2019) { Time.zone.local(2019, 2, 1) }
+  let(:april2019) { Time.zone.local(2019, 4, 1) }
+  let(:data) do
+    {
+      role: 'Teaching Assistant',
+      organisation: Faker::Educator.secondary_school,
+      details: Faker::Lorem.paragraph_by_chars(number: 300),
+      commitment: %w[full_time part_time].sample,
+      relevant_skills: [true, true, true, false].sample,
+    }
+  end
+
+  let(:application_form_with_no_breaks) do
+    create(:application_form) do |form|
+      data[:organisation] = 'Vararu School'
+      data[:start_date] = Time.zone.local(2019, 1, 1)
+      data[:end_date] = Time.zone.local(2019, 6, 1)
+      form.application_work_experiences.create(data)
+
+      data[:organisation] = 'Theo School'
+      data[:start_date] = Time.zone.local(2019, 6, 1)
+      data[:end_date] = Time.zone.local(2019, 8, 1)
+      form.application_work_experiences.create(data)
+
+      data[:organisation] = 'Vararu School'
+      data[:start_date] = Time.zone.local(2019, 8, 1)
+      data[:end_date] = Time.zone.local(2019, 10, 1)
+      form.application_work_experiences.create(data)
```

Comment:
I think we should just build 3 objects here rather than using the form above data to create it.